plugins/labhub.py

```diff
@@ -222,6 +222,18 @@ def create_issue_cmd(self, msg, match):
                 target=user,
             )
 
+    @staticmethod
+    def profile_link(msg):
+        gitter_username = msg.frm.nick
+        pattern = '^(.*)(_.*?)$'
+        if '_gitlab' in gitter_username:
+            username = re.match(
+                pattern, gitter_username.split('@')[-1]).group(1)
+            return 'https://gitlab.com/{user}'.format(user=username)
+        else:
```

Comment:
Isin't there login through twitter option too on Gitter? You need to get that too imo.

Reply:
@abhishalya IMO not a lot of users use twitter so that could be left out for now.

Reply:
No, again, that should be a part of this issue itself.